aiohttp/cookiejar.py

```diff
@@ -37,6 +39,17 @@ def __init__(self, *, unsafe=False, loop=None):
         self._next_expiration = ceil(self._loop.time())
         self._expirations = {}
 
+    def load(self, file_path):
+        f = open(file_path, 'rb')
```

Comment:
Please support both `str` and `pathlib.Path` types.
Just add `file_path = pathlib.Path(file_path)` and use [Path.open](https://docs.python.org/3/library/pathlib.html#pathlib.Path.open) API.